app/assets/stylesheets/pages/experience_stage.sass

```diff
@@ -25,9 +25,27 @@
     &:hover,&:focus,&.active
       @extend .text-experience
 
-
 #experience_stage-content
   padding: 20px 0px
   background-color: $lightestGrayBG
   &.affix
-    margin-top: 40px
\ No newline at end of file
+    margin-top: 40px
+  .order-by
+    padding: 0px
+  .badge
+    @extend .text-default, .text-small
+    +border-radius(2em)
+    background-color: transparent
+    vertical-align: bottom
+    .headline
+      font-size: $font-size-small
```

Comment:
Don't think we'll need this as I declared a text-xsmall class on the actual badge.